app/build.gradle.kts

```diff
@@ -126,8 +134,11 @@ dependencies {
         implementation(libs.androidx.appcompat)
         implementation(libs.material)
         implementation(libs.androidx.lifecycle.runtime.ktx)
+    implementation(libs.androidx.junit.ktx)
+    implementation(libs.ui.test.junit4.android)
+    testImplementation(libs.androidx.espresso.intents)
 
```

Comment:
Conflict with main branch: Unresolved merge between testing dependencies and navigation dependencies. Please merge keeping both `androidx.junit.ktx`, `ui.test.junit4.android`, `espresso.intents`, and `navigation.runtime.ktx` as they serve different purposes in the app.